extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -1884,16 +1878,41 @@ export class ProjectsController {
 
 		// Move the file
 		try {
-			const project = await Project.openProject(projectUri.fsPath);
-
-			// TODO: swap out with DacFx projects apis - currently moving pre/post deploy scripts don't work, but they should work after the swap
-			await fse.move(sourceFileNode.fileSystemUri.fsPath, newPath!);
-			await project.exclude(project.files.find(f => f.fsUri.fsPath === sourceFileNode.fileSystemUri.fsPath)!);
-			await project.addExistingItem(newPath!);
+			await this.move(sourceFileNode, projectUri.fsPath, newPath, sourceFileNode.relativeProjectUri.fsPath);
 		} catch (e) {
 			void vscode.window.showErrorMessage(constants.errorMovingFile(sourceFileNode.fileSystemUri.fsPath, newPath, utils.getErrorMessage(e)));
 		}
 	}
+
+	/**
+	 * Moves a file to a different location
+	 * @param node Node being moved
+	 * @param projectFilePath Full file path to .sqlproj
+	 * @param destinationRelativePath path of the destination, relative to .sqlproj
+	 * @param originalRelativePath path of the original location, relative to .sqlproj
+	 */
+	private async move(node: BaseProjectTreeItem, projectFilePath: string, destinationRelativePath: string, originalRelativePath: string): Promise<void> {
```

Comment:
Does `node` not contain `originalRelativePath` in it somehow?

Reply:
good point - I cleaned that up to get the original path from node